### DISCUSSION TITLE
3596823A-F096-4E0C-8F82-6FC72E457BC5

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#80] [General]
# 3596823A-F096-4E0C-8F82-6FC72E457BC5

Luhman 16 is a binary brown dwarf system about 6.50 light-years away in the constellation Vela, containing the nearest brown dwarfs to Sol. The system seems to be fairly young, with an age of about 560 million years. Observations using the Very Large Telescope have allowed the creation of a map of Luhman 16 B. The textures included here are fictional, but based on real data.

https://celestia.mobi/addon?item=3596823A-F096-4E0C-8F82-6FC72E457BC5&lang=en

**Comment by exocritic5457** [comment]
There should honestly be more addons like this with star textures based on actual data. Stars like HD 12545, Zeta Andromedae, Speedy Mic, and so on. Feels relatively untouched, and it's pretty cool.

**Reply by TheLostProbe** [reply]
there's a few more of them but I think only 2 (this one and Pi1 Gruis) are on celestia.mobi, the rest are on the Forums somewhere
